pkg/signup/signup_service.go

```diff
@@ -17,6 +17,11 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	SignupReasonNoCondition  = "UserSignupComplete condition not set"
```

Comment:
Condition Reason is supposed to be one word. Something like `userSignCompleteCondtionNotSet` but in practice the missing condition means that the UserSignup resource is not handled by the controller yet. It basically equals `PendingApproval`.

Reply:
Changed to `PendingApproval`